### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by gonzagamax**
Hello, here is my work.

### CODE REVIEW

**Review by maxgonzaga — CHANGES_REQUESTED** [review]
I think you it must be improved.

**Review by gonzagamax — COMMENTED** [review]
i did the requested changes.

**Comment by maxgonzaga** [comment]
Not good yest.

File: test-pr.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-This is a test!
```
Comment (gonzagamax) [inline test-pr.txt]: I think this should be pt-br.

**Pull request merged**